src/gui/MainWidget.cpp

```diff
@@ -154,6 +154,7 @@ MainWidget::MainWidget(QPair<QStringList, QStringList> &QMPArguments)
 	tray = new QSystemTrayIcon(this);
 	tray->setIcon(QMPlay2Core.getIconFromTheme("QMPlay2-panel", QMPlay2Core.getQMPlay2Pixmap()));
 	tray->setVisible(settings.getBool("TrayVisible", true));
+	QApplication::instance()->setProperty("system_tray", QVariant::fromValue<void *>(tray));
```

Comment:
Is `QApplication::instance()` better than `qApp` macro?
I think that you can create another variable for tray in `QMPlay2Core` class (without including `QSystemTrayIcon`). API/ABI compatibility is now not important.